src/version.h.in

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#ifndef VERSION_STORE
+    #error "Include only from versionstore.cpp do avoid unnecessary recompile on every commit."
+#endif
+
+#define MIXXX_VERSION "@MIXXX_VERSION@"
```

Comment:
I think we should also add the individual CMAKE_PROJECT_VERSION components here (major, minor, tweak). Then we can later use them for determining the manual URLs, etc., instead of duplicating the information.